SharedCode/Controller/PokemonDetailController.cs

```diff
@@ -35,6 +36,11 @@ public async void LoadPokemonImage(int pokeId)
             var image = await Repository.GetPokemonImage(pokeId);
             viewListener.updatePokemonImage(image);
         }
+
+        public async void GetEvolutionChainByPokemonId(int id)
+        {
+            var data = await Repository.GetEvolutionChainByPokemonId(id);
```

Comment:
maybe you can call the method to update the view instead of just setting it in a var

Reply:
I think this should be modified when fetching the pokemon details.

Reply:
ok